Extensions/IISWebAppDeploy/Src/Tasks/IISWebAppDeploy/MsDeployOnTargetMachines.ps1

```diff
@@ -184,6 +189,6 @@ function Execute-Main
     Write-Verbose "AdditionalArguments = $AdditionalArguments"
 
     $overRideParams = Compute-MsDeploy-SetParams -websiteName $websiteName -overRideParams $overRideParams
-    Deploy-Website -webDeployPkg $WebDeployPackage -webDeployParamFile $WebDeployParamFile -overRiderParams $OverRideParams -websiteName $websiteName -excludeFilesFromAppData $excludeFilesFromAppData -removeAdditionalFiles $removeAdditionalFiles -takeAppOffline $takeAppOffline -additionalArguments $AdditionalArguments
```

Comment:
Debugging of the task is becoming critical due to missing of verbose logs in few methods. Can you accommodate them.